src/Chat/utils/index.js

```diff
@@ -15,4 +15,12 @@ const getMessageQueueWeight = ({
   return constants.RATE_LIMIT_USER
 }
 
-export { getMessageQueueWeight }
+const defer = cb => {
+  if (typeof self.setImmediate !== 'undefined') {
```

Comment:
I think I am missing something. What is `self` here?

Reply:
It's in case people want to use `twitch-js` in WebWorkers (https://developer.mozilla.org/en-US/docs/Web/API/Window/self)

Reply:
TIL ❤️